toc/working-groups/foundational-infrastructure.md

```diff
@@ -59,6 +59,8 @@ bots:
   github: bbr-ci
 - name: mysql-ci
   github: pcf-core-services-writer
+- name: tas-operability-bot
+  github: tas-operability-bot
```

Comment:
Could you move this bot under the relevant area? This is now supported by the working group automation (example [here](https://github.com/cloudfoundry/community/blob/main/toc/working-groups/app-runtime-interfaces.md?plain=1#L324-L326)).